<launch>
   <node pkg="cluster_extraction" type="cluster_extraction" name="cluster_extraction" output="screen"/>
   <node pkg="cluster_extraction" type="cluster_transform" name="cluster_transform" output="screen"/>
   <node pkg="cluster_extraction" type="manipulator_controller" name="manipulator_controller" output="screen"/>
</launch>
